openstack/sharedfilesystems/v2/sharenetworks/results.go

```diff
@@ -28,15 +31,36 @@ type ShareNetwork struct {
 	// The Share Network description
 	Description string `json:"description"`
 	// The date and time stamp when the Share Network was created
-	CreatedAt string `json:"created_at"`
+	CreatedAt gophercloud.JSONRFC3339MilliNoZ `json:"created_at"`
 	// The date and time stamp when the Share Network was updated
-	UpdatedAt string `json:"updated_at"`
+	UpdatedAt gophercloud.JSONRFC3339MilliNoZ `json:"updated_at"`
 }
 
 type commonResult struct {
 	gophercloud.Result
 }
 
+// ShareNetworkPage is a pagination.pager that is returned from a call to the List function.
+type ShareNetworkPage struct {
+	pagination.LinkedPageBase
```

Comment:
It's probably this: https://github.com/gophercloud/gophercloud/blob/master/pagination/marker.go#L19

For that, you'll need to create a `LastMarker` method and a `NextPageURL` that sets `offset` in the query string instead of `marker`.